packages/server/test/support/fixtures/projects/e2e/static/settimeout_redirect_document_href.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Document</title>
+</head>
+<body>
+  <script type="text/javascript">
+    setTimeout(() => {
+      document.location.href = 'index.html';
+    }, 1)
```

Comment:
why are all redirects after `setTimeout`?

Reply:
that seems to be the condition for this class of bug, it only happens when you do the setter inside of a setTimeout